### PR TITLE
chore: update support channel

### PR DESCRIPTION
**Pull request by rudolfs**
Closes https://github.com/radicle-dev/radicle-upstream/issues/1938.

Screenshot 2021-09-08 at 15 55 18

### CODE REVIEW

**Comment by brandonhaslegs** [comment]
"Get in touch directly" doesn't seem like a link for radicle.community

**Comment by brandonhaslegs** [comment]
Suggestion:

"View the community forums" for the radicle.community link

and

"Get in touch directly" for the Discord link

File: ui/App/SettingsScreen.svelte [hunk 1]
```diff
@@ -247,19 +247,16 @@
         <div class="section-item">
           <p class="typo-text-bold">Get in touch directly</p>
```
Comment (brandonhaslegs) [inline ui/App/SettingsScreen.svelte:248]:
```suggestion
          <p class="typo-text-bold"> View community forum</p>
```

File: ui/App/SettingsScreen.svelte [hunk 2]
```diff
@@ -247,19 +247,16 @@
...
-              class="typo-link"
-              href="https://radicle.community/c/site-feedback/2">
-              radicle.community
-            </a>
+            <ExternalLink url="https://radicle.community/c/site-feedback/2"
+              >radicle.community</ExternalLink>
           </div>
         </div>
         <div class="section-item">
           <p class="typo-text-bold">Join the community chat</p>
```
Comment (brandonhaslegs) [inline ui/App/SettingsScreen.svelte:255]:
```suggestion
          <p class="typo-text-bold"> Get in touch directly</p>
```

**Comment by rudolfs** [comment]
Updated the code and the screenshot. PTAL!

**Review by brandonhaslegs — APPROVED** [review]

**Pull request merged**